### PR TITLE
Make more screen size independent.

### PR DESCRIPTION
**Pull request by DavidShoe**

### CODE REVIEW

File: IoTBrowser/IoTBrowser/MainPage.xaml [hunk 1]
```diff
@@ -36,21 +36,33 @@
...
+            <Button Grid.Column="2" x:Name="Go_Web" Content="Go!" HorizontalAlignment="Right" VerticalAlignment="Center" Height="60" Width="100" Click="Go_Web_Click"/>
+        </Grid>
         
         <!--Web view control-->
         <WebView x:Name="webView" Grid.Row="1"/>
         
         <!--Favorites buttons-->
-        <StackPanel Grid.Row="2" Orientation="Horizontal" >
-            <Button x:Name="Go_WOD" VerticalAlignment="Center" HorizontalAlignment="Center" Height="60" Width="120" Margin="0,0,15,0" Click="Go_WOD_Click">
+        <Grid Grid.Row="2" HorizontalAlignment="Center">
```
Comment (bfjelds) [inline IoTBrowser/IoTBrowser/MainPage.xaml]: should we put this into some kind of scroll viewer...there may be screens of smaller width.

Comment (DavidShoe) [inline IoTBrowser/IoTBrowser/MainPage.xaml]: It already handles putting scrollbars for the content.

Comment (bfjelds) [inline IoTBrowser/IoTBrowser/MainPage.xaml]: i'm talking about the buttons ... not the content

Comment (DavidShoe) [inline IoTBrowser/IoTBrowser/MainPage.xaml]: The buttons will fit in anything bigger than 400 px, smaller than that and the usefulness of this sample is pretty much shot anyway.

Comment (bfjelds) [inline IoTBrowser/IoTBrowser/MainPage.xaml]: I don't think that the usefulness of the browser is as important as enabling the buttons to click for any screen size is.  The buttons allow someone to trace through the code as a page changes (wihtout a keyboard attached, right?).  Maybe it isn't vital, but if we are making changes to accommodate smaller screens, I don't know why we'd reinforce 400 as a minimum.

**Pull request merged**